aiven/client/argx.py

```diff
@@ -196,47 +260,100 @@ def print_response(
         file=None,
     ):  # pylint: disable=redefined-builtin
         """print request response in chosen format"""
-        if file is None:
-            file = sys.stdout
+
+        # Deprecation compatibility: map the old keyword args into the new enum:
+        output_stream = self._output_stream if file is None else file
+
+        output_format = self._output_format
+
+        if json is True:
```

Comment:
Is there a reason to check for `True` explicitly?

Reply:
`json` isn't just "kind of truthy" it should distinctly be True or False. This encourages test suites to fail if some undefined behavior is being used (better would be for mypy to just catch it but we're not there yet).